lib/rspec/core/memoized_helpers.rb

```diff
@@ -1,3 +1,5 @@
+require 'thread'
```

Comment:
> Means 1.8 won't be threadsafe, but 1.8's been end of lifed, so shouldn't be any new development on it, anyway.

Well, if you use the stdlib `Mutex` if it's available, then wouldn't 1.8 still be threadsafe?  After all, if the thread stdlib hasn't been loaded than the user can't be using multiple threads, right?